auth0/v3/management/__init__.py

```diff
@@ -27,11 +28,13 @@
 from .users import Users
 from .users_by_email import UsersByEmail
 
+
```

Comment:
Could you delete this line return (and the one in auth0.py), it's causing isort to [fail the build ](https://app.circleci.com/pipelines/github/auth0/auth0-python/684/workflows/4ab3b99a-f986-4b9a-b66e-5b69e245d140/jobs/2027)
<img width="1357" alt="image" src="https://user-images.githubusercontent.com/1299658/189856538-bf138717-4011-46d2-ac32-e7c1e4cb1d03.png">